lua/nvim-web-devicons/icons-default.lua

```diff
@@ -992,12 +1382,30 @@ local icons_by_file_extension = {
     cterm_color = "199",
     name = "GraphQL",
   },
+  ["gresource"] = {
+    icon = "",
+    color = "#ffffff",
+    cterm_color = "231",
+    name = "GTK",
+  },
+  ["gv"] = {
+    icon = "󱁉",
+    color = "#30638e",
+    cterm_color = "24",
+    name = "Gv",
+  },
   ["h"] = {
     icon = "",
     color = "#a074c4",
     cterm_color = "140",
     name = "H",
   },
+  ["hc"] = {
+    icon = "󰂢",
+    color = "#f6fe59",
+    cterm_color = "227",
+    name = "HolyC",
+  },
```

Comment:
Please remove this one. TempleOS is controversial.

Reply:
ok